src/help/components/entry/Entry.tsx

```diff
@@ -76,6 +76,59 @@ const cleanTextOptions: IOptions = {
   },
 };
 
+export const HelpEntryContent = ({
+  data,
+  isStale,
+  handleClick,
+}: {
+  data: HelpEntryResponse;
+  isStale?: boolean;
+  handleClick: MouseEventHandler<HTMLElement>;
+}) => {
+  const [lastModifed, html] = useMemo(
+    () =>
+      data?.content
+        ? [
+            parseDate(data.lastModified),
+            cleanText(marked(data.content), cleanTextOptions),
+          ]
+        : [],
+    [data]
+  );
+
+  if (!html) {
+    return <ErrorHandler />;
+  }
+
+  return (
+    <>
+      <h1 className={data.categories.includes('faq') ? 'big' : undefined}>
+        {data.title}
+      </h1>
```

Comment:
I have concerns about heading hierarchy in a side panel not following the flow of the document, but would need to read more on it though, I don't know what is the good way to do things here